lib/main.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
Can i extends other function to get the job done except StatelessWidget?

Reply:
Since Flutter is a widget-based framework of Dart language, primarily two types of widgets exist in Flutter called statelessWidget and statefulWidget. These are the base of building Flutter app.